src/ComponentHelper.js

```diff
@@ -69,7 +69,24 @@ module.exports.findDirtyComponents = function(el, syncedComps, cachedData) {
     }
 
     var oldCompData = cachedData[compKey];
-    if (!deepEqual(oldCompData, newCompData)) {
+    if (compKey.endsWith("---position---") || compKey === "position"
```

Comment:
This doesn't seem right, we should be checking agaisnt the actual coponent names not trying to reverse it out of the key we generated from those component names.